src/components/structures/SpaceRoomView.tsx

```diff
@@ -158,7 +162,33 @@ const onBetaClick = () => {
     });
 };
 
-const SpacePreview = ({ space, onJoinButtonClicked, onRejectButtonClicked }) => {
+// XXX: temporary community migration component
```

Comment:
What does the XXX denote here? I'm used to using it for far more egregious things! Does it mean - TODO we should delete this in future?

Reply:
pretty much, more of a tech note for temporary things